### PR TITLE
index: Add platforms description and logo

### PR DESCRIPTION
**Pull request by mattiasbus**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -2,13 +2,21 @@
...
+Host Mobility provides a complete Linux platform with driver support for all hardware interfaces and with a customizable distribution.
+
+All hardware interfaces are accessible via well defined APIs. We use the the standard Linux way of doing things as much as possible. This way the platform environment is familiar to developers who has worked with embedded Linux in their past.
 
 | Name                                      | Description  |
 | -----                                     | ------------ |
 | [Host Monitor X](hardware/host-monitor-x) |              |
-| [MX4-T30FR](hardware/mx4-t30fr)           |              |
-| [MX4-C61](hardware/mx4-c61)               |              |
+| [MX4-T30FR](hardware/mx-4-t30fr)           |              |
```
Comment (RikardO-HM) [inline docs/index.md:18]: where is the hardware folder?

Comment (RikardO-HM) [reply docs/index.md:18]: found it on #9

**Review by RikardO-HM — DISMISSED** [review]

File: docs/index.md [hunk 2]
```diff
@@ -2,13 +2,21 @@
...
-# Hardware
+![Host Mobility](https://hostmobility-eng.setek.se/wp-content/uploads/2021/06/logo-host-mobility200.png)
+
+## The Host Mobility Platforms
+
+Host Mobility designs and manufactures small computing devices that can be connected to vehicles of any kind, fetch diagnostics from a long range of data buses, register the current location, transmit data for further analysis, and much more – all using standard Linux APIs. You are free to run any application on top of our hardware platforms.
+
+Host Mobility provides a complete Linux platform with driver support for all hardware interfaces and with a customizable distribution.
+
+All hardware interfaces are accessible via well defined APIs. We use the the standard Linux way of doing things as much as possible. This way the platform environment is familiar to developers who has worked with embedded Linux in their past.
```
Comment (albsod) [inline docs/index.md:13]:
```suggestion
All hardware interfaces are accessible via well-defined APIs. We prefer the standard Linux way of doing things as much as possible. This way the platform environment is familiar to developers who have worked with embedded Linux in the past.
```

**Comment by albsod** [comment]
We should use a higher-resolution logo. There are plenty available....

File: docs/index.md (hunk 2 above)
Comment (RikardO-HM) [reply docs/index.md]: the text is removed is it as it should @mattiasbus ?

File: docs/index.md [hunk 3]
```diff
@@ -2,31 +2,35 @@
 title: Index
 ---
 
-# Hardware
+![Host Mobility](https://hostmobility-eng.setek.se/wp-content/uploads/2021/06/logo-host-mobility200.png)
```
Comment (RikardO-HM) [inline docs/index.md:5]: only issue is this logo. should be replaced

**Review by RikardO-HM — APPROVED** [review]

**Pull request merged**